src/System.Net.Primitives/src/System/Net/CookieContainer.cs

```diff
@@ -940,7 +940,7 @@ internal string GetCookieHeader(Uri uri, out string optCookie2)
             CookieCollection cookies = InternalGetCookies(uri);
             string delimiter = string.Empty;
 
-            var builder = new StringBuilder();
```

Comment:
Same as above. `var` is recommended in situations where the type of the local is present on the RHS of the initialization expression.